apps/wing/src/commands/compile.ts

```diff
@@ -95,6 +95,7 @@ export async function compile(entrypoint: string, options: CompileOptions): Prom
   log("work dir: %s", workDir);
 
   process.env["WING_SYNTH_DIR"] = tmpSynthDir;
+  process.env["WING_FILE_DIR"] = resolve(wingDir);
```

Comment:
I'm good with that. I thought a little bit about this and initially avoided it because we don't really have the concept of a project (yet), but file_dir is pretty silly.